### DISCUSSION TITLE
Overriding CHROME_START_WIDTH by providing a custom settings file did not work

### DISCUSSION THREAD

**Discussion by maburrub** [seleniumbase/SeleniumBase#1275] [Q&A] [answered]
I tried to change the browser width by directly updating the settings.py file, it worked
I tried to override the default value by changing CHROME_START_WIDTH in a custom settings file using --settings-file=FILE did not work. what am I missing?

**Comment by mdmintz** [comment, marked as answer]
Thank you for reporting this. It is a bug, and I'll get it fixed.
I created an issue ticket for this:
https://github.com/seleniumbase/SeleniumBase/issues/1276

**Reply by mdmintz** [reply]
The fix was released in https://github.com/seleniumbase/SeleniumBase/releases/tag/v2.4.35
(``2.4.35``)
